dhctl/cmd/dhctl/main.go

```diff
@@ -66,6 +66,8 @@ func main() {
 		return
 	}
 
+	commands.DefineMirrorCommand(kpApp)
```

Comment:
Just move this above line 62, and remove line 64. My git somehow skipped commit that does this.